internal/components/components_test.go

```diff
@@ -25,6 +25,7 @@ import (
 
 func TestDefaultComponents(t *testing.T) {
 	expectedExtensions := []config.Type{
+		"ecs_observer",
```

Comment:
If we are comfortable adding this (cc @jrcamp, @dmitryax) it needs to be documented: https://github.com/signalfx/splunk-otel-collector/blob/main/docs/components.md

Reply:
Should be good to add. But yes, it needs to be added to the doc

Reply:
I added ecs_observer to components doc